orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTaskSpec.groovy

```diff
@@ -1156,7 +1156,7 @@ class CreateBakeTaskSpec extends Specification {
   }
 
   @Unroll
-  def "sets previouslyBaked flag to #previouslyBaked when status is #status.state"() {
+  def "sets previouslyBaked flag to #previouslyBaked when status is #status"() {
```

Comment:
Is this really better?  From what I can see:
before: `sets previouslyBaked flag to false when status is RUNNING`
after: `sets previouslyBaked flag to false when status is com.netflix.spinnaker.orca.bakery.api.BakeStatus(id:186c951c-179f-4625-ab7d-ac39b0a8d093, state:RUNNING, result:null, resourceId:null)	`

Reply:
Agreed. `#status.state` gives more precise info, going to revert this change.

Reply:
Done